L1Trigger/TrackTrigger/python/TrackQualityParams_cfi.py

```diff
@@ -1,7 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-TrackQualityParams = cms.PSet(qualityAlgorithm = cms.string("GBDT_cpp"), #None, Cut, NN, GBDT, GBDT_cpp
-                              # This emulation GBDT is optimised for the HYBRID_NEWKF emulation and works with the emulation of the KF out module
+TrackQualityParams = cms.PSet(# This emulation GBDT is optimised for the HYBRID_NEWKF emulation and works with the emulation of the KF out module
                               # It is compatible with the HYBRID simulation and will give equivilant performance with this workflow
                               ONNXmodel = cms.FileInPath("L1Trigger/TrackTrigger/data/L1_TrackQuality_GBDT_emulation_digitized.json"),
```

Comment:
As we now use a .json for the model do we want to change the naming convention throughout to move away from "ONNXmodel" to prevent confusion.  We don't actually call the ONNX runtime anywhere so do we still need to include it at the top of the L1TrackQuality.h?

Reply:
Good idea, I'll change the naming convention.